### PR TITLE
Added if check for less than php8 with libxml_disable_entity_loader function

### PR DESCRIPTION
**Pull request by DHouweling**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ website/error_log
 # Ignore directories generated by Composer
 vendor/
 composer.lock
+.idea/
```
Comment (BertvanHoekelen) [inline .gitignore]: This should not be added to the repo but should be [ignored globally](https://stackoverflow.com/questions/59328900/how-to-globally-ignore-the-idea-folder-in-git-not-just-adding-it-to-gitignor).

File: library/AuthenticationResponse.php [hunk 2]
```diff
@@ -55,7 +55,10 @@ public static function getException($exception) {
     }
     
     public static function parse($xml) {
-        $oldValue = libxml_disable_entity_loader(true);
+
+        if (\PHP_VERSION_ID < 80000) {
+            $oldValue = libxml_disable_entity_loader(true);
+        }
```
Comment (BertvanHoekelen) [inline library/AuthenticationResponse.php]:
```suggestion
        if (\PHP_VERSION_ID < 80000) {
            $oldValue = libxml_disable_entity_loader(true);
        }

```

File: library/DirectoryResponse.php [hunk 3]
```diff
@@ -58,7 +58,9 @@ public static function getException($exception) {
     }
 
     public static function parse($xml) {
-        $oldValue = libxml_disable_entity_loader(true);
+        if (\PHP_VERSION_ID < 80000) {
+            $oldValue = libxml_disable_entity_loader(true);
+        }
```
Comment (BertvanHoekelen) [inline library/DirectoryResponse.php:63]:
```suggestion
        }

```

File: library/StatusResponse.php [hunk 4]
```diff
@@ -80,7 +80,9 @@ public static function getException($exception) {
     }
     
     public static function parse(Configuration $config, $xml) {
-        $oldValue = libxml_disable_entity_loader(true);
+        if (\PHP_VERSION_ID < 80000) {
+            $oldValue = libxml_disable_entity_loader(true);
+        }
```
Comment (BertvanHoekelen) [inline library/StatusResponse.php:85]:
```suggestion
        }

```

File: library/XmlProcessor.php [hunk 5]
```diff
@@ -186,11 +186,15 @@ private function checkIdxSignature(Configuration $config, $doc) {
     }
     
     public function verifySignature(Configuration $config, $xml) {
-        $oldValue = libxml_disable_entity_loader(true);
+        if (\PHP_VERSION_ID < 80000) {
+            $oldValue = libxml_disable_entity_loader(true);
+        }
```
Comment (BertvanHoekelen) [inline library/XmlProcessor.php:191]:
```suggestion
        }

```

File: library/XmlProcessor.php [hunk 6]
```diff
@@ -201,11 +205,15 @@ public function verifySignature(Configuration $config, $xml) {
     }
 
     public function verifySchema($xml) {
-        $oldValue = libxml_disable_entity_loader(true);
+        if (\PHP_VERSION_ID < 80000) {
+            $oldValue = libxml_disable_entity_loader(true);
+        }
```
Comment (BertvanHoekelen) [inline library/XmlProcessor.php:211]:
```suggestion
        }

```

**Review by BertvanHoekelen — CHANGES_REQUESTED** [review]
~Also add php 8.0 to composer.json~

Already > 7.3 so 8.0 is already supported.

**Comment by dvdmunckhof** [comment]
@grigoresc it'd be highly appreciated if you could merge this

**Review by dvdmunckhof — APPROVED** [review]

**Comment by grigoresc** [comment]
Thank you for your changes!

**Review by grigoresc — APPROVED** [review]

**Pull request merged**